src/index.js

```diff
@@ -2,11 +2,13 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
+import Login from './Login';
 import * as serviceWorker from './serviceWorker';
```

Comment:
If we've decided that Login should be a child of App, we should remove line 5, correct?